corehq/apps/users/forms.py

```diff
@@ -1389,18 +1416,35 @@ def __init__(self, *args, **kwargs):
             ),
             crispy.Field("search_string", data_bind="value: search_string"),
         ]
+        user_type_title = 'Users'
+
         if self.user_type == MOBILE_USER_TYPE:
+            user_type_title = 'Mobile Workers'
             fields += [
                 crispy.Div(
-                    crispy.Field("location_id", data_bind="value: location_id"),
-                    data_bind="slideVisible: !isCrossDomain()",
+                    crispy.Field(
+                        "location_id",
+                        data_bind="value: location_id",
+                    ),
+                    data_bind="slideVisible: !isCrossDomain(), event: {change: location_change}",
+                ),
+                crispy.Div(
+                    crispy.Field(
+                        "selected_location_only",
+                        data_bind="checked: selected_location_only"
+                    ),
+                    data_bind="slideVisible: !isCrossDomain() && location_id",
+                ),
+                crispy.Field(
+                    "user_active_status",
+                    data_bind="value: user_active_status",
                 ),
                 crispy.Field("columns", data_bind="value: columns"),
             ]
 
         self.helper.layout = crispy.Layout(
             crispy.Fieldset(
-                _("Filter and Download Users"),
+                _(f'Filter and Download {user_type_title}'),
```

Comment:
This won't be translated - the string passed to `_` can't be dynamic. Something like `_("Filter and Download {user_type}").format(user_type=user_type_title)` would work. The user type titles themselves also need to be translated above.

Reply:
Oh, this is new to me. Thanks for clarifying. I always thought the string will be evaluated first, then the translation happens, but seems to me that's not the case. After reading up a little now it actually makes more sense this way, since with arbitrary values the translation will not perform as expected :)